lib/Github/Api/PullRequest/Comments.php

```diff
@@ -11,9 +11,13 @@
  */
 class Comments extends AbstractApi
 {
-    public function all($username, $repository, $pullRequest)
+    public function all($username, $repository, $pullRequest = null)
     {
-        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($pullRequest).'/comments');
+        if (null !== $pullRequest) {
+            return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($pullRequest).'/comments');
+        }
+
+        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/comments');
```

Comment:
This change is not really a part of the PR, right? 

Let's keep it anyways (for this time)